src/onnx/parse_dequantizelinear.cpp

```diff
@@ -0,0 +1,49 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_dequantizelinear : op_parser<parse_dequantizelinear>
+{
+    std::vector<op_desc> operators() const { return {{"DequantizeLinear"}}; }
+
+    instruction_ref parse(const op_desc& /*opd*/,
+                          const onnx_parser& /*parser*/,
+                          const onnx_parser::node_info& info,
+                          std::vector<instruction_ref> args) const
+    {
+        int axis = 1;
+        if(contains(info.attributes, "axis"))
+            axis = info.attributes.at("axis").i();
+
+        auto input_lens = args[0]->get_shape().lens();
+
+        auto sub_zero_point = args[0];
+
+        if(args.size() == 3)
+        {
+            auto zero_point = args[2];
+            if(axis != 1)
```

Comment:
Looks like the broadcasting here has some issues.
In my understanding, if it is a scalar shape, we can use multibroadcast; 
if it is a 1D tensor, it is a broadcast as make_op("broadcast", {{"axis", axis}, {"dims", input_lens}}) (not dependent on the axis value).

Reply:
Another way for the broadcasting can be as:
For scalar shape, just do multibroadcasting, this is straightforward.
For the 1D case, we can first do a reshape as for the 1D tensor as {1, 1, (axis_dim use the dim of the 1D-tensor), 1), then do a multibroadcasting....